File: Go/fizzbuzz2.go
package main

import (
	"fmt"
)

const LIMIT = 100

func main() {
	for i := 1; i <= LIMIT; i++ {
		switch {
		case i%15 == 0:
			fmt.Print("FizzBuzz")
		case i%3 == 0:
			fmt.Print("Fizz")
		case i%5 == 0:
			fmt.Print("Buzz")
		default:
			fmt.Print(i)
		}
		if i < LIMIT {
			fmt.Print(", ")
		}
	}
	fmt.Println()
}

File: Perl/fizzbuzz2.pl
$LIMIT = 100;

for ($i = 1; $i <= $LIMIT; $i++) {
	$r = $i % 15;
	if ($r == 0) {
		print "FizzBuzz";
	} elsif ($r == 3 || $r == 6 || $r == 9 || $r == 12) {
		print "Fizz";
	} elsif ($r == 5 || $r == 10) {
		print "Buzz";
	} else {
		print $i;
	}
	if ($i < $LIMIT) {
		print ", ";
	}
}

print "\n";
